README.md

```diff
@@ -68,3 +68,37 @@ topk(100, instance:etcd_mvcc_db_total_size_in_use_in_bytes:sum)
 ```console
 sum without (_id) (quantile by (_id)(0.5, instance:etcd_disk_backend_commit_duration_seconds:histogram_quantile{quantile="0.99"}) + on(_id) group_left(provider) (topk by (_id) (1, id_provider*0)))
 ```
+
+## Tests
+
+This repository is compatible with the [OpenShift Tests Extension (OTE)](https://github.com/openshift-eng/openshift-tests-extension) framework.
+
+### Building the test binary
+
+```bash
+make build
+```
+
+### Running test suites and tests
+
+```bash
+# Run a specific test suite or test
+./cluster-etcd-operator-tests-ext run-suite SUITE=openshift/cluster-etcd-operator/all
+./cluster-etcd-operator-tests-ext run-test "test-name"
+
+# Run with JUnit output
+./cluster-etcd-operator-tests-ext run-suite SUITE=openshift/cluster-etcd-operator/all --junit-path=/tmp/junit-results/junit.xml
```

Comment:
It is different from https://github.com/openshift/cluster-kube-apiserver-operator/blob/main/README.md?plain=1#L200